graph_annotation.go

```diff
@@ -0,0 +1,18 @@
+package mackerel
+
+// GraphAnnotation parameters to post graph annotation
+type GraphAnnotation struct {
+	Service     string   `json:"service,omitempty"`
+	Roles       []string `json:"roles,omitempty"`
+	From        int64    `json:"from,omitempty"`
```

Comment:
I assumed time.Time will be marshaled as int64, but it doesn't.... https://play.golang.org/p/-6ptCnvZwL
Maybe I mistook time.Time for time.Duration.

It's somewhat tedious to convert it internally, so maybe we can  leave it as int64.